docs/sophia_stdlib.md

```diff
@@ -470,38 +470,6 @@ Chain.block_height : int"
 The height of the current block (i.e. the block in which the current call will be included).
 
 
-##### coinbase
-```
-Chain.coinbase : address
-```
-
-The address of the account that mined the current block.
-
-
-##### timestamp
-```
-Chain.timestamp : int
-```
-
-The timestamp of the current block (unix time, milliseconds).
-
-
-##### difficulty
-```
-Chain.difficulty : int
-```
-
-The difficulty of the current block.
-
-
-##### gas
-```
-Chain.gas_limit : int
```

Comment:
has this been removed or did it never exist? 🤔

Reply:
No, just too little coffee...